frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -24,3 +27,6 @@ export const getCephPVs = (pvsData: K8sResourceKind[] = []): K8sResourceKind[] =
 
 export const getCephNodes = (nodesData: K8sResourceKind[] = []): K8sResourceKind[] =>
   nodesData.filter((node) => _.keys(_.get(node, 'metadata.labels')).includes(cephStorageLabel));
+
+export const getCephSC = (scData: K8sResourceKind[]): K8sResourceKind[] =>
+  scData.filter((sc) => cephStorageProvisioners.includes(_.get(sc, 'provisioner')));
```

Comment:
Nit: No need to have brackets around `sc` parameter.

Reply:
@gnehapk automatically added by linter.